Error:
```
2020-10-18T20:41:55.1174150Z [ 41%] Building CXX object CMakeFiles/libaol.dir/modules/aol/platformDependent.cpp.obj
2020-10-18T20:41:55.1545532Z platformDependent.cpp
2020-10-18T20:41:56.2541707Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:56.6094792Z %SRC_DIR%\quocmesh\modules\aol\platformDependent.cpp(256): error C2664: 'DWORD GetTempPathA(DWORD,LPSTR)': cannot convert argument 2 from 'WCHAR [1024]' to 'LPSTR'
2020-10-18T20:41:56.6097593Z %SRC_DIR%\quocmesh\modules\aol\platformDependent.cpp(257): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style cast
2020-10-18T20:41:56.6102958Z %SRC_DIR%\quocmesh\modules\aol\platformDependent.cpp(265): error C2664: 'UINT GetTempFileNameA(LPCSTR,LPCSTR,UINT,LPSTR)': cannot convert argument 1 from 'WCHAR [1024]' to 'LPCSTR'
2020-10-18T20:41:56.6104688Z %SRC_DIR%\quocmesh\modules\aol\platformDependent.cpp(268): note: Types pointed to are unrelated; conversion requires 
```
@conda-forge/match-series any idea?

<details close><summary>Build log</summary><p>

```
2020-10-18T20:36:22.8297748Z ##[section]Starting: Build recipe
2020-10-18T20:36:22.8500535Z ==============================================================================
2020-10-18T20:36:22.8502056Z Task         : Command line
2020-10-18T20:36:22.8502492Z Description  : Run a command line script using Bash on Linux and macOS and cmd.exe on Windows
2020-10-18T20:36:22.8503011Z Version      : 2.176.1
2020-10-18T20:36:22.8503310Z Author       : Microsoft Corporation
2020-10-18T20:36:22.8503843Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/command-line
2020-10-18T20:36:22.8504438Z ==============================================================================
2020-10-18T20:36:23.9477679Z Generating script.
2020-10-18T20:36:23.9988045Z ========================== Starting Command Output ===========================
2020-10-18T20:36:24.0327329Z ##[command]"C:\windows\system32\cmd.exe" /D /E:ON /V:OFF /S /C "CALL "D:\a\_temp\465f0a0c-6aa8-4cda-9352-3b3885155b94.cmd""
2020-10-18T20:36:25.7951674Z No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
2020-10-18T20:36:25.7953697Z WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
2020-10-18T20:36:25.8371975Z INFO:conda_build.variants:Adding in variants from internal_defaults
2020-10-18T20:36:25.8372908Z Adding in variants from internal_defaults
2020-10-18T20:36:25.8374011Z INFO:conda_build.variants:Adding in variants from .ci_support\win_64_.yaml
2020-10-18T20:36:25.8374729Z Adding in variants from .ci_support\win_64_.yaml
2020-10-18T20:36:26.0236090Z INFO:conda_build.metadata:Attempting to finalize metadata for match-series
2020-10-18T20:36:26.0237107Z Attempting to finalize metadata for match-series
2020-10-18T20:37:38.8465721Z Collecting package metadata (repodata.json): ...working... done
2020-10-18T20:37:42.3639764Z Solving environment: ...working... done
2020-10-18T20:38:12.0762989Z Collecting package metadata (repodata.json): ...working... done
2020-10-18T20:38:19.5875717Z Solving environment: ...working... done
2020-10-18T20:38:35.8826896Z Collecting package metadata (repodata.json): ...working... done
2020-10-18T20:38:46.0169471Z Solving environment: ...working... done
2020-10-18T20:38:46.0569049Z BUILD START: ['match-series-1.0.0-h71e8331_6.tar.bz2']
2020-10-18T20:38:56.3593775Z Collecting package metadata (repodata.json): ...working... done
2020-10-18T20:39:06.3957863Z Solving environment: ...working... done
2020-10-18T20:39:06.5627541Z 
2020-10-18T20:39:06.5628697Z ## Package Plan ##
2020-10-18T20:39:06.5629257Z 
2020-10-18T20:39:06.5629798Z   environment location: D:\bld\match-series_1603053385828\_h_env
2020-10-18T20:39:06.5630324Z 
2020-10-18T20:39:06.5637787Z 
2020-10-18T20:39:06.5638613Z The following NEW packages will be INSTALLED:
2020-10-18T20:39:06.5639275Z 
2020-10-18T20:39:06.5640076Z     bzip2:                   1.0.8-he774522_3          conda-forge
2020-10-18T20:39:06.5640906Z     fftw:                    3.3.8-nompi_h24e91a8_1112 conda-forge
2020-10-18T20:39:06.5641524Z     jpeg:                    9d-he774522_0             conda-forge
2020-10-18T20:39:06.5642372Z     libblas:                 3.8.0-11_h8933c1f_netlib  conda-forge
2020-10-18T20:39:06.5643108Z     libcblas:                3.8.0-11_h8933c1f_netlib  conda-forge
2020-10-18T20:39:06.5644020Z     liblapack:               3.8.0-11_h8933c1f_netlib  conda-forge
2020-10-18T20:39:06.5645120Z     liblapacke:              3.8.0-11_h8933c1f_netlib  conda-forge
2020-10-18T20:39:06.5645910Z     libpng:                  1.6.37-ha81a0f5_2         conda-forge
2020-10-18T20:39:06.5646753Z     libtiff:                 4.1.0-h885aae3_6          conda-forge
2020-10-18T20:39:06.5647683Z     lz4-c:                   1.9.2-h62dcd97_2          conda-forge
2020-10-18T20:39:06.5648551Z     m2w64-gcc-libgfortran:   5.3.0-6                   defaults   
2020-10-18T20:39:06.5649330Z     m2w64-gcc-libs:          5.3.0-7                   defaults   
2020-10-18T20:39:06.5651764Z     m2w64-gcc-libs-core:     5.3.0-7                   defaults   
2020-10-18T20:39:06.5652617Z     m2w64-gmp:               6.1.0-2                   defaults   
2020-10-18T20:39:06.5653479Z     m2w64-libwinpthread-git: 5.0.0.4634.697f757-2      defaults   
2020-10-18T20:39:06.5654391Z     msys2-conda-epoch:       20160418-1                defaults   
2020-10-18T20:39:06.5655181Z     suitesparse:             5.4.0-h5d0cbe0_1          conda-forge
2020-10-18T20:39:06.5656299Z     vc:                      14.1-h869be7e_1           conda-forge
2020-10-18T20:39:06.5657024Z     vs2015_runtime:          14.16.27012-h30e32a0_2    conda-forge
2020-10-18T20:39:06.5657921Z     xz:                      5.2.5-h62dcd97_1          conda-forge
2020-10-18T20:39:06.5658715Z     zlib:                    1.2.11-h62dcd97_1010      conda-forge
2020-10-18T20:39:06.5659579Z     zstd:                    1.4.5-h1f3a1b7_2          conda-forge
2020-10-18T20:39:06.5660088Z 
2020-10-18T20:39:08.7734474Z Preparing transaction: ...working... done
2020-10-18T20:39:10.9826741Z Verifying transaction: ...working... done
2020-10-18T20:39:16.0192834Z Executing transaction: ...working... done
2020-10-18T20:39:28.6630150Z Collecting package metadata (repodata.json): ...working... done
2020-10-18T20:39:32.0321843Z Solving environment: ...working... done
2020-10-18T20:39:43.9540566Z Collecting package metadata (repodata.json): ...working... done
2020-10-18T20:39:54.1096097Z Solving environment: ...working... done
2020-10-18T20:39:54.2917192Z 
2020-10-18T20:39:54.2918524Z ## Package Plan ##
2020-10-18T20:39:54.2919303Z 
2020-10-18T20:39:54.2920169Z   environment location: D:\bld\match-series_1603053385828\_build_env
2020-10-18T20:39:54.2920791Z 
2020-10-18T20:39:54.2921523Z 
2020-10-18T20:39:54.2922258Z The following NEW packages will be INSTALLED:
2020-10-18T20:39:54.2922800Z 
2020-10-18T20:39:54.2923460Z     cmake:          3.18.2-h001a745_0      conda-forge
2020-10-18T20:39:54.2924235Z     vs2015_runtime: 14.16.27012-h30e32a0_2 conda-forge
2020-10-18T20:39:54.2925150Z     vs2017_win-64:  19.16.27038-h2e3bad8_2 conda-forge
2020-10-18T20:39:54.2925874Z     vswhere:        2.7.1-h21ff451_0       defaults   
2020-10-18T20:39:54.2926374Z 
2020-10-18T20:39:54.7454623Z Preparing transaction: ...working... done
2020-10-18T20:39:56.9106373Z Verifying transaction: ...working... done
2020-10-18T20:40:16.2651481Z Executing transaction: ...working... done
2020-10-18T20:40:16.2818162Z INFO:conda_build.source:Source cache directory is: D:\bld\src_cache
2020-10-18T20:40:16.2819412Z INFO conda_build.source:download_to_cache(44): Source cache directory is: D:\bld\src_cache
2020-10-18T20:40:16.2820340Z Source cache directory is: D:\bld\src_cache
2020-10-18T20:40:16.2822692Z INFO:conda_build.source:Downloading source to cache: v1.0.0_9e74ca6bc5.tar.gz
2020-10-18T20:40:16.2823922Z Downloading source to cache: v1.0.0_9e74ca6bc5.tar.gz
2020-10-18T20:40:16.2824944Z INFO conda_build.source:download_to_cache(69): Downloading source to cache: v1.0.0_9e74ca6bc5.tar.gz
2020-10-18T20:40:16.2826194Z INFO:conda_build.source:Downloading https://github.com/berkels/match-series/archive/v1.0.0.tar.gz
2020-10-18T20:40:16.2827457Z INFO conda_build.source:download_to_cache(83): Downloading https://github.com/berkels/match-series/archive/v1.0.0.tar.gz
2020-10-18T20:40:16.2828502Z Downloading https://github.com/berkels/match-series/archive/v1.0.0.tar.gz
2020-10-18T20:40:16.8384659Z INFO:conda_build.source:Success
2020-10-18T20:40:16.8385879Z Success
2020-10-18T20:40:16.8387271Z INFO conda_build.source:download_to_cache(94): Success
2020-10-18T20:40:16.8449469Z Extracting download
2020-10-18T20:40:19.4282661Z source tree in: D:\bld\match-series_1603053385828\work
2020-10-18T20:40:19.8281840Z Rewriting env in output: {'BUILD_PREFIX': 'D:\\bld\\match-series_1603053385828\\_build_env',
2020-10-18T20:40:19.8283103Z  'PREFIX': 'D:\\bld\\match-series_1603053385828\\_h_env',
2020-10-18T20:40:19.8284203Z  'SRC_DIR': 'D:\\bld\\match-series_1603053385828\\work'}
2020-10-18T20:40:19.8664643Z 
2020-10-18T20:40:19.8666323Z (base) %SRC_DIR%>IF "" == "" (call %SRC_DIR%\build_env_setup.bat ) 
2020-10-18T20:40:19.8678232Z 
2020-10-18T20:40:19.8679046Z (base) %SRC_DIR%>set "CONDA_BUILD=1" 
2020-10-18T20:40:19.8683090Z 
2020-10-18T20:40:19.8684007Z (base) %SRC_DIR%>set "PYTHONNOUSERSITE=1" 
2020-10-18T20:40:19.8690471Z 
2020-10-18T20:40:19.8691278Z (base) %SRC_DIR%>set "CONDA_DEFAULT_ENV=%PREFIX%" 
2020-10-18T20:40:19.8696932Z 
2020-10-18T20:40:19.8697948Z (base) %SRC_DIR%>set "ARCH=64" 
2020-10-18T20:40:19.8704041Z 
2020-10-18T20:40:19.8705009Z (base) %SRC_DIR%>set "PREFIX=%PREFIX%" 
2020-10-18T20:40:19.8710420Z 
2020-10-18T20:40:19.8711101Z (base) %SRC_DIR%>set "BUILD_PREFIX=%BUILD_PREFIX%" 
2020-10-18T20:40:19.8716833Z 
2020-10-18T20:40:19.8718123Z (base) %SRC_DIR%>set "SYS_PREFIX=C:\Miniconda" 
2020-10-18T20:40:19.8723803Z 
2020-10-18T20:40:19.8724798Z (base) %SRC_DIR%>set "SYS_PYTHON=C:\Miniconda\python.exe" 
2020-10-18T20:40:19.8729863Z 
2020-10-18T20:40:19.8730728Z (base) %SRC_DIR%>set "SUBDIR=win-64" 
2020-10-18T20:40:19.8737366Z 
2020-10-18T20:40:19.8738110Z (base) %SRC_DIR%>set "build_platform=win-64" 
2020-10-18T20:40:19.8743883Z 
2020-10-18T20:40:19.8744566Z (base) %SRC_DIR%>set "SRC_DIR=%SRC_DIR%" 
2020-10-18T20:40:19.8750244Z 
2020-10-18T20:40:19.8751059Z (base) %SRC_DIR%>set "ROOT=C:\Miniconda" 
2020-10-18T20:40:19.8757340Z 
2020-10-18T20:40:19.8758326Z (base) %SRC_DIR%>set "CONDA_PY=36" 
2020-10-18T20:40:19.8764359Z 
2020-10-18T20:40:19.8765336Z (base) %SRC_DIR%>set "PY3K=1" 
2020-10-18T20:40:19.8771208Z 
2020-10-18T20:40:19.8771988Z (base) %SRC_DIR%>set "PY_VER=3.6" 
2020-10-18T20:40:19.8778536Z 
2020-10-18T20:40:19.8780889Z (base) %SRC_DIR%>set "STDLIB_DIR=%PREFIX%\Lib" 
2020-10-18T20:40:19.8785545Z 
2020-10-18T20:40:19.8786742Z (base) %SRC_DIR%>set "SP_DIR=%PREFIX%\Lib\site-packages" 
2020-10-18T20:40:19.8793352Z 
2020-10-18T20:40:19.8794502Z (base) %SRC_DIR%>set "NPY_VER=1.11" 
2020-10-18T20:40:19.8799280Z 
2020-10-18T20:40:19.8799945Z (base) %SRC_DIR%>set "CONDA_NPY=1.11" 
2020-10-18T20:40:19.8805735Z 
2020-10-18T20:40:19.8806518Z (base) %SRC_DIR%>set "NPY_DISTUTILS_APPEND_FLAGS=1" 
2020-10-18T20:40:19.8812154Z 
2020-10-18T20:40:19.8813109Z (base) %SRC_DIR%>set "PERL_VER=5.26" 
2020-10-18T20:40:19.8818558Z 
2020-10-18T20:40:19.8819328Z (base) %SRC_DIR%>set "CONDA_PERL=5.26.2" 
2020-10-18T20:40:19.8825318Z 
2020-10-18T20:40:19.8826403Z (base) %SRC_DIR%>set "LUA_VER=5" 
2020-10-18T20:40:19.8831852Z 
2020-10-18T20:40:19.8832619Z (base) %SRC_DIR%>set "CONDA_LUA=5" 
2020-10-18T20:40:19.8838277Z 
2020-10-18T20:40:19.8839121Z (base) %SRC_DIR%>set "R_VER=3.4" 
2020-10-18T20:40:19.8844016Z 
2020-10-18T20:40:19.8844812Z (base) %SRC_DIR%>set "CONDA_R=3.4" 
2020-10-18T20:40:19.8851513Z 
2020-10-18T20:40:19.8854964Z (base) %SRC_DIR%>set "PKG_NAME=match-series" 
2020-10-18T20:40:19.8857086Z 
2020-10-18T20:40:19.8862358Z (base) %SRC_DIR%>set "PKG_VERSION=1.0.0" 
2020-10-18T20:40:19.8871777Z 
2020-10-18T20:40:19.8872566Z (base) %SRC_DIR%>set "PKG_BUILDNUM=6" 
2020-10-18T20:40:19.8875863Z 
2020-10-18T20:40:19.8876857Z (base) %SRC_DIR%>set "PKG_BUILD_STRING=placeholder" 
2020-10-18T20:40:19.8882168Z 
2020-10-18T20:40:19.8883252Z (base) %SRC_DIR%>set "PKG_HASH=1234567" 
2020-10-18T20:40:19.8888574Z 
2020-10-18T20:40:19.8889666Z (base) %SRC_DIR%>set "RECIPE_DIR=D:\a\1\s\recipe" 
2020-10-18T20:40:19.8895443Z 
2020-10-18T20:40:19.8896240Z (base) %SRC_DIR%>set "CPU_COUNT=2" 
2020-10-18T20:40:19.8908447Z 
2020-10-18T20:40:19.8915029Z (base) %SRC_DIR%>set "SHLIB_EXT=.dll" 
2020-10-18T20:40:19.8915518Z 
2020-10-18T20:40:19.8924276Z (base) %SRC_DIR%>set "PATH=%BUILD_PREFIX%;%BUILD_PREFIX%\Library\mingw-w64\bin;%BUILD_PREFIX%\Library\usr\bin;;%BUILD_PREFIX%\Library\bin;%BUILD_PREFIX%\Scripts;%BUILD_PREFIX%\Scripts;%PREFIX%;%PREFIX%\Library\mingw-w64\bin;%PREFIX%\Library\usr\bin;;%PREFIX%\Library\bin;%PREFIX%\Scripts;%PREFIX%\Scripts;C:\Miniconda;C:\Miniconda\Library\mingw-w64\bin;C:\Miniconda\Library\usr\bin;C:\Miniconda\Library\bin;C:\Miniconda\Scripts;C:\Miniconda\bin;C:\Miniconda\condabin;C:\Miniconda\Scripts;C:\Miniconda;C:\agents\2.175.2\externals\git\cmd;C:\Users\VssAdministrator\.dotnet\tools;C:\Program Files\Mercurial;C:\Program Files\MongoDB\Server\4.4\bin;C:\ProgramData\kind;C:\vcpkg;C:\cf-cli;C:\Program Files (x86)\NSIS;C:\Program Files\Mercurial;C:\h\windows\stack\2.3.3\x64;C:\ProgramData\chocolatey\lib\ghc.8.10.2\tools\ghc-8.10.2\bin;C:\Program Files\dotnet;C:\mysql-5.7.21-winx64\bin;C:\Program Files\R\R-4.0.2\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\h\windows\go\1.14.9\x64\bin;C:\h\windows\Python\3.7.9\x64\Scripts;C:\h\windows\Python\3.7.9\x64;C:\h\windows\Ruby\2.5.8\x64\bin;C:\Program Files\Java\jdk8u265-b01\bin;C:\aliyun-cli;C:\npm\prefix;C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin;C:\Program Files\Microsoft MPI\Bin;C:\windows\system32;C:\windows;C:\windows\System32\Wbem;C:\windows\System32\WindowsPowerShell\v1.0;C:\Program Files\Microsoft\Web Platform Installer;C:\Program Files\Docker;C:\Program Files\PowerShell\7;C:\Program Files\dotnet;C:\Program Files\Microsoft SQL Server\130\Tools\Binn;C:\Program Files (x86)\Microsoft SQL Server\110\DTS\Binn;C:\Program Files (x86)\Microsoft SQL Server\120\DTS\Binn;C:\Program Files (x86)\Microsoft SQL Server\130\DTS\Binn;C:\Program Files (x86)\Microsoft SQL Server\140\DTS\Binn;C:\Program Files (x86)\Microsoft SQL Server\150\DTS\Binn;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.6.3\bin;C:\Program Files\TortoiseSVN\bin;C:\SeleniumWebDrivers\ChromeDriver;C:\SeleniumWebDrivers\EdgeDriver;C:\Program Files\Amazon\AWSCLIV2;C:\Program Files\Amazon\SessionManagerPlugin\bin;C:\Program Files\Amazon\AWSSAMCLI\bin;C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\bin;C:\Users\VssAdministrator\AppData\Local\Microsoft\WindowsApps" 
2020-10-18T20:40:19.8937051Z 
2020-10-18T20:40:19.8939603Z (base) %SRC_DIR%>set "SCRIPTS=%PREFIX%\Scripts" 
2020-10-18T20:40:19.8940151Z 
2020-10-18T20:40:19.8947102Z (base) %SRC_DIR%>set "LIBRARY_PREFIX=%PREFIX%\Library" 
2020-10-18T20:40:19.8949092Z 
2020-10-18T20:40:19.8949696Z (base) %SRC_DIR%>set "LIBRARY_BIN=%PREFIX%\Library\bin" 
2020-10-18T20:40:19.8950279Z 
2020-10-18T20:40:19.8950917Z (base) %SRC_DIR%>set "LIBRARY_INC=%PREFIX%\Library\include" 
2020-10-18T20:40:19.8951524Z 
2020-10-18T20:40:19.8952132Z (base) %SRC_DIR%>set "LIBRARY_LIB=%PREFIX%\Library\lib" 
2020-10-18T20:40:19.8952925Z 
2020-10-18T20:40:19.8953481Z (base) %SRC_DIR%>set "CYGWIN_PREFIX=/cygdrive/d/bld/match-series_1603053385828/_h_env" 
2020-10-18T20:40:19.8954174Z 
2020-10-18T20:40:19.8954687Z (base) %SRC_DIR%>set "ALLUSERSPROFILE=C:\ProgramData" 
2020-10-18T20:40:19.8955100Z 
2020-10-18T20:40:19.8955956Z (base) %SRC_DIR%>set "APPDATA=C:\Users\VssAdministrator\AppData\Roaming" 
2020-10-18T20:40:19.8956755Z 
2020-10-18T20:40:19.8957503Z (base) %SRC_DIR%>set "CommonProgramFiles=C:\Program Files\Common Files" 
2020-10-18T20:40:19.8963065Z 
2020-10-18T20:40:19.8964318Z (base) %SRC_DIR%>set "CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files" 
2020-10-18T20:40:19.8974081Z 
2020-10-18T20:40:19.8975972Z (base) %SRC_DIR%>set "CommonProgramW6432=C:\Program Files\Common Files" 
2020-10-18T20:40:19.8978607Z 
2020-10-18T20:40:19.8979372Z (base) %SRC_DIR%>set "COMPUTERNAME=WIN-E45VE44RLHA" 
2020-10-18T20:40:19.8983167Z 
2020-10-18T20:40:19.8985154Z (base) %SRC_DIR%>set "ComSpec=C:\windows\system32\cmd.exe" 
2020-10-18T20:40:19.8987478Z 
2020-10-18T20:40:19.8991850Z (base) %SRC_DIR%>set "HOMEDRIVE=C:" 
2020-10-18T20:40:19.8992851Z 
2020-10-18T20:40:19.8993706Z (base) %SRC_DIR%>set "HOMEPATH=\Users\VssAdministrator" 
2020-10-18T20:40:19.8998442Z 
2020-10-18T20:40:19.8999604Z (base) %SRC_DIR%>set "LOCALAPPDATA=C:\Users\VssAdministrator\AppData\Local" 
2020-10-18T20:40:19.9005443Z 
2020-10-18T20:40:19.9006538Z (base) %SRC_DIR%>set "LOGONSERVER=\\WIN-E45VE44RLHA" 
2020-10-18T20:40:19.9008059Z 
2020-10-18T20:40:19.9376546Z (base) %SRC_DIR%>set "NUMBER_OF_PROCESSORS=4" 
2020-10-18T20:40:19.9377126Z 
2020-10-18T20:40:19.9377840Z (base) %SRC_DIR%>set "PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.CPL" 
2020-10-18T20:40:19.9378412Z 
2020-10-18T20:40:19.9379203Z (base) %SRC_DIR%>set "ProgramData=C:\ProgramData" 
2020-10-18T20:40:19.9379918Z 
2020-10-18T20:40:19.9380592Z (base) %SRC_DIR%>set "ProgramFiles=C:\Program Files" 
2020-10-18T20:40:19.9381233Z 
2020-10-18T20:40:19.9382564Z (base) %SRC_DIR%>set "ProgramFiles(x86)=C:\Program Files (x86)" 
2020-10-18T20:40:19.9383256Z 
2020-10-18T20:40:19.9384194Z (base) %SRC_DIR%>set "ProgramW6432=C:\Program Files" 
2020-10-18T20:40:19.9384772Z 
2020-10-18T20:40:19.9385159Z (base) %SRC_DIR%>set "PROMPT=(base) $P$G" 
2020-10-18T20:40:19.9385691Z 
2020-10-18T20:40:19.9388666Z (base) %SRC_DIR%>set "PSModulePath=C:\Users\VssAdministrator\Documents\WindowsPowerShell\Modules;C:\Modules\azurerm_2.1.0;C:\Modules\azure_2.1.0;C:\Users\packer\Documents\WindowsPowerShell\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\windows\system32\WindowsPowerShell\v1.0\Modules;C:\Program Files\Microsoft SQL Server\130\Tools\PowerShell\Modules\;C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\platform\PowerShell" 
2020-10-18T20:40:19.9390178Z 
2020-10-18T20:40:19.9390772Z (base) %SRC_DIR%>set "PUBLIC=C:\Users\Public" 
2020-10-18T20:40:19.9391376Z 
2020-10-18T20:40:19.9391808Z (base) %SRC_DIR%>set "SystemDrive=C:" 
2020-10-18T20:40:19.9392214Z 
2020-10-18T20:40:19.9392610Z (base) %SRC_DIR%>set "SystemRoot=C:\windows" 
2020-10-18T20:40:19.9393007Z 
2020-10-18T20:40:19.9394677Z (base) %SRC_DIR%>set "TEMP=C:\Users\VSSADM~1\AppData\Local\Temp" 
2020-10-18T20:40:19.9395354Z 
2020-10-18T20:40:19.9396036Z (base) %SRC_DIR%>set "TMP=C:\Users\VSSADM~1\AppData\Local\Temp" 
2020-10-18T20:40:19.9396768Z 
2020-10-18T20:40:19.9397258Z (base) %SRC_DIR%>set "USERDOMAIN=WIN-E45VE44RLHA" 
2020-10-18T20:40:19.9397709Z 
2020-10-18T20:40:19.9398344Z (base) %SRC_DIR%>set "USERNAME=VssAdministrator" 
2020-10-18T20:40:19.9398973Z 
2020-10-18T20:40:19.9399993Z (base) %SRC_DIR%>set "USERPROFILE=C:\Users\VssAdministrator" 
2020-10-18T20:40:19.9400804Z 
2020-10-18T20:40:19.9401361Z (base) %SRC_DIR%>set "windir=C:\windows" 
2020-10-18T20:40:19.9401862Z 
2020-10-18T20:40:19.9402258Z (base) %SRC_DIR%>set "PROCESSOR_ARCHITECTURE=AMD64" 
2020-10-18T20:40:19.9402485Z 
2020-10-18T20:40:19.9402807Z (base) %SRC_DIR%>set "PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 85 Stepping 4, GenuineIntel" 
2020-10-18T20:40:19.9403097Z 
2020-10-18T20:40:19.9404847Z (base) %SRC_DIR%>set "BUILD=amd64-pc-windows-19.0.0" 
2020-10-18T20:40:19.9405444Z 
2020-10-18T20:40:19.9406031Z (base) %SRC_DIR%>set "VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\" 
2020-10-18T20:40:19.9406564Z 
2020-10-18T20:40:19.9407704Z (base) %SRC_DIR%>set "pin_run_as_build=OrderedDict([('python', OrderedDict([('min_pin', 'x.x'), ('max_pin', 'x.x')])), ('r-base', OrderedDict([('min_pin', 'x.x'), ('max_pin', 'x.x')])), ('bzip2', {'max_pin': 'x'}), ('fftw', {'max_pin': 'x'}), ('libpng', {'max_pin': 'x.x'}), ('libtiff', {'max_pin': 'x'}), ('zlib', {'max_pin': 'x.x'})])" 
2020-10-18T20:40:19.9408940Z 
2020-10-18T20:40:19.9410326Z (base) %SRC_DIR%>set "ignore_build_only_deps={'numpy', 'python'}" 
2020-10-18T20:40:19.9410859Z 
2020-10-18T20:40:19.9411469Z (base) %SRC_DIR%>set "extend_keys={'pin_run_as_build', 'ignore_build_only_deps', 'ignore_version', 'extend_keys'}" 
2020-10-18T20:40:19.9411981Z 
2020-10-18T20:40:19.9414258Z (base) %SRC_DIR%>set "suitesparse=5.4" 
2020-10-18T20:40:19.9414680Z 
2020-10-18T20:40:19.9415178Z (base) %SRC_DIR%>set "target_platform=win-64" 
2020-10-18T20:40:19.9415561Z 
2020-10-18T20:40:19.9416063Z (base) %SRC_DIR%>set "fortran_compiler=gfortran" 
2020-10-18T20:40:19.9417084Z 
2020-10-18T20:40:19.9417541Z (base) %SRC_DIR%>set "r_base=3.4" 
2020-10-18T20:40:19.9417979Z 
2020-10-18T20:40:19.9418412Z (base) %SRC_DIR%>set "libtiff=4.1.0" 
2020-10-18T20:40:19.9418809Z 
2020-10-18T20:40:19.9419546Z (base) %SRC_DIR%>set "bzip2=1" 
2020-10-18T20:40:19.9420021Z 
2020-10-18T20:40:19.9420546Z (base) %SRC_DIR%>set "cran_mirror=https://cran.r-project.org" 
2020-10-18T20:40:19.9421201Z 
2020-10-18T20:40:19.9421509Z (base) %SRC_DIR%>set "libpng=1.6" 
2020-10-18T20:40:19.9421840Z 
2020-10-18T20:40:19.9424078Z (base) %SRC_DIR%>set "c_compiler=vs2017" 
2020-10-18T20:40:19.9424619Z 
2020-10-18T20:40:19.9425018Z (base) %SRC_DIR%>set "fftw=3" 
2020-10-18T20:40:19.9425379Z 
2020-10-18T20:40:19.9425843Z (base) %SRC_DIR%>set "cxx_compiler=vs2017" 
2020-10-18T20:40:19.9426243Z 
2020-10-18T20:40:19.9426699Z (base) %SRC_DIR%>set "zlib=1.2" 
2020-10-18T20:40:19.9427177Z 
2020-10-18T20:40:19.9427733Z (base) %SRC_DIR%>set "libblas=3.8 *netlib" 
2020-10-18T20:40:19.9428090Z 
2020-10-18T20:40:19.9428526Z (base) %SRC_DIR%>set "CI=azure" 
2020-10-18T20:40:19.9428969Z 
2020-10-18T20:40:19.9429355Z (base) %SRC_DIR%>set "vc=14" 
2020-10-18T20:40:19.9429841Z 
2020-10-18T20:40:19.9430357Z (base) %SRC_DIR%>set "channel_sources=conda-forge,defaults" 
2020-10-18T20:40:19.9430760Z 
2020-10-18T20:40:19.9431257Z (base) %SRC_DIR%>set "cpu_optimization_target=nocona" 
2020-10-18T20:40:19.9431774Z 
2020-10-18T20:40:19.9433914Z (base) %SRC_DIR%>set "channel_targets=conda-forge main" 
2020-10-18T20:40:19.9434352Z 
2020-10-18T20:40:19.9434870Z (base) %SRC_DIR%>set "CONDA_BUILD_STATE=BUILD" 
2020-10-18T20:40:19.9435360Z 
2020-10-18T20:40:19.9435902Z (base) %SRC_DIR%>set "PIP_NO_BUILD_ISOLATION=False" 
2020-10-18T20:40:19.9436289Z 
2020-10-18T20:40:19.9436768Z (base) %SRC_DIR%>set "PIP_NO_DEPENDENCIES=True" 
2020-10-18T20:40:19.9437219Z 
2020-10-18T20:40:19.9437708Z (base) %SRC_DIR%>set "PIP_IGNORE_INSTALLED=True" 
2020-10-18T20:40:19.9438149Z 
2020-10-18T20:40:19.9438843Z (base) %SRC_DIR%>set "PIP_CACHE_DIR=D:\bld\match-series_1603053385828\pip_cache" 
2020-10-18T20:40:19.9439513Z 
2020-10-18T20:40:19.9439867Z (base) %SRC_DIR%>set "PIP_NO_INDEX=True" 
2020-10-18T20:40:19.9440124Z 
2020-10-18T20:40:19.9440788Z (base) %SRC_DIR%>set "INCLUDE=%PREFIX%\Library\include;" 
2020-10-18T20:40:19.9441317Z 
2020-10-18T20:40:19.9441740Z (base) %SRC_DIR%>set "LIB=%PREFIX%\Library\lib;" 
2020-10-18T20:40:19.9444031Z 
2020-10-18T20:40:19.9444795Z (base) %SRC_DIR%>call "C:\Miniconda\Scripts\..\condabin\conda_hook.bat" 
2020-10-18T20:40:19.9445274Z 
2020-10-18T20:40:19.9445863Z (base) %SRC_DIR%>call "C:\Miniconda\Scripts\..\condabin\conda.bat" activate "%PREFIX%" 
2020-10-18T20:40:20.2666464Z 
2020-10-18T20:40:20.2667993Z (%PREFIX%) %SRC_DIR%>call "C:\Miniconda\Scripts\..\condabin\conda.bat" activate --stack "%BUILD_PREFIX%" 
2020-10-18T20:40:20.6509089Z 
2020-10-18T20:40:20.6509908Z %SRC_DIR%>SET DISTUTILS_USE_SDK=1 
2020-10-18T20:40:20.6516609Z 
2020-10-18T20:40:20.6517611Z %SRC_DIR%>SET MSSdk=1 
2020-10-18T20:40:20.6524562Z 
2020-10-18T20:40:20.6525082Z %SRC_DIR%>SET "VS_VERSION=15.0" 
2020-10-18T20:40:20.6539197Z 
2020-10-18T20:40:20.6540680Z %SRC_DIR%>SET "VS_MAJOR=15" 
2020-10-18T20:40:20.6570544Z 
2020-10-18T20:40:20.6571333Z %SRC_DIR%>SET "VS_YEAR=2017" 
2020-10-18T20:40:20.6572035Z 
2020-10-18T20:40:20.6573352Z %SRC_DIR%>set "MSYS2_ARG_CONV_EXCL=/AI;/AL;/OUT;/out" 
2020-10-18T20:40:20.6594221Z 
2020-10-18T20:40:20.6595058Z %SRC_DIR%>set "MSYS2_ENV_CONV_EXCL=CL" 
2020-10-18T20:40:20.6595670Z 
2020-10-18T20:40:20.6596223Z %SRC_DIR%>set "PY_VCRUNTIME_REDIST=%PREFIX%\bin\vcruntime140.dll" 
2020-10-18T20:40:20.6599640Z 
2020-10-18T20:40:20.6600370Z %SRC_DIR%>set "CXX=cl.exe" 
2020-10-18T20:40:20.6630823Z 
2020-10-18T20:40:20.6642264Z %SRC_DIR%>set "CC=cl.exe" 
2020-10-18T20:40:20.6642839Z 
2020-10-18T20:40:20.6643272Z %SRC_DIR%>set "VSINSTALLDIR=" 
2020-10-18T20:40:20.6643719Z 
2020-10-18T20:40:20.6644446Z %SRC_DIR%>for /F "usebackq tokens=*" %i in (`vswhere.exe -nologo -products * -version [15.0,16.0) -property installationPath`) do (set "VSINSTALLDIR=%i\" ) 
2020-10-18T20:40:20.8887905Z 
2020-10-18T20:40:20.8890325Z %SRC_DIR%>(set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\" ) 
2020-10-18T20:40:20.8902093Z 
2020-10-18T20:40:20.8907629Z %SRC_DIR%>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\" (for /F "usebackq tokens=*" %i in (`vswhere.exe -nologo -products * -requires Microsoft.VisualStudio.Component.VC.v141.x86.x64 -property installationPath`) do (set "VSINSTALLDIR=%i\" ) ) 
2020-10-18T20:40:20.8908549Z 
2020-10-18T20:40:20.8923587Z %SRC_DIR%>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\" ) 
2020-10-18T20:40:20.8928575Z 
2020-10-18T20:40:20.8931793Z %SRC_DIR%>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\" ) 
2020-10-18T20:40:20.8942992Z 
2020-10-18T20:40:20.8944192Z %SRC_DIR%>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\" ) 
2020-10-18T20:40:20.8949138Z 
2020-10-18T20:40:20.8953169Z %SRC_DIR%>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\" ) 
2020-10-18T20:40:20.8961140Z 
2020-10-18T20:40:20.8961803Z %SRC_DIR%>IF NOT "1" == "" (
2020-10-18T20:40:20.8962500Z set "INCLUDE=%PREFIX%\Library\include;%PREFIX%\Library\include;"  
2020-10-18T20:40:20.8963548Z  set "LIB=%PREFIX%\Library\lib;%PREFIX%\Library\lib;"  
2020-10-18T20:40:20.8964621Z  set "CMAKE_PREFIX_PATH=%PREFIX%\Library;" 
2020-10-18T20:40:20.8965299Z ) 
2020-10-18T20:40:20.8976228Z 
2020-10-18T20:40:20.8977240Z %SRC_DIR%>call :GetWin10SdkDir 
2020-10-18T20:40:20.8993516Z 
2020-10-18T20:40:20.8994649Z %SRC_DIR%>call :GetWin10SdkDirHelper HKLM\SOFTWARE\Wow6432Node  1>nul 2>&1 
2020-10-18T20:40:20.9287796Z 
2020-10-18T20:40:20.9289207Z %SRC_DIR%>if errorlevel 1 call :GetWin10SdkDirHelper HKCU\SOFTWARE\Wow6432Node  1>nul 2>&1 
2020-10-18T20:40:20.9290154Z 
2020-10-18T20:40:20.9290710Z %SRC_DIR%>if errorlevel 1 call :GetWin10SdkDirHelper HKLM\SOFTWARE  1>nul 2>&1 
2020-10-18T20:40:20.9293277Z 
2020-10-18T20:40:20.9298427Z %SRC_DIR%>if errorlevel 1 call :GetWin10SdkDirHelper HKCU\SOFTWARE  1>nul 2>&1 
2020-10-18T20:40:20.9301036Z 
2020-10-18T20:40:20.9302755Z %SRC_DIR%>if errorlevel 1 exit /B 1 
2020-10-18T20:40:20.9303525Z 
2020-10-18T20:40:20.9304359Z %SRC_DIR%>exit /B 0 
2020-10-18T20:40:20.9311141Z 
2020-10-18T20:40:20.9313309Z %SRC_DIR%>for /F %i in ('dir /ON /B "C:\Program Files (x86)\Windows Kits\10\\include\10.*"') DO (SET WindowsSDKVer=%~i ) 
2020-10-18T20:40:20.9612426Z 
2020-10-18T20:40:20.9613765Z %SRC_DIR%>(SET WindowsSDKVer=10.0.10240.0 ) 
2020-10-18T20:40:20.9615358Z 
2020-10-18T20:40:20.9615961Z %SRC_DIR%>(SET WindowsSDKVer=10.0.10586.0 ) 
2020-10-18T20:40:20.9619042Z 
2020-10-18T20:40:20.9620170Z %SRC_DIR%>(SET WindowsSDKVer=10.0.14393.0 ) 
2020-10-18T20:40:20.9620989Z 
2020-10-18T20:40:20.9622696Z %SRC_DIR%>(SET WindowsSDKVer=10.0.15063.0 ) 
2020-10-18T20:40:20.9623699Z 
2020-10-18T20:40:20.9624494Z %SRC_DIR%>(SET WindowsSDKVer=10.0.16299.0 ) 
2020-10-18T20:40:20.9627045Z 
2020-10-18T20:40:20.9628622Z %SRC_DIR%>(SET WindowsSDKVer=10.0.17134.0 ) 
2020-10-18T20:40:20.9630105Z 
2020-10-18T20:40:20.9630874Z %SRC_DIR%>(SET WindowsSDKVer=10.0.17763.0 ) 
2020-10-18T20:40:20.9634631Z 
2020-10-18T20:40:20.9636171Z %SRC_DIR%>if errorlevel 1 (echo "Didn't find any windows 10 SDK. I'm not sure if things will work, but let's try..." )  else (echo Windows SDK version found as: "10.0.17763.0" ) 
2020-10-18T20:40:20.9639281Z Windows SDK version found as: "10.0.17763.0"
2020-10-18T20:40:20.9639951Z 
2020-10-18T20:40:20.9640974Z %SRC_DIR%>IF "win-64" == "win-64" (
2020-10-18T20:40:20.9641885Z set "CMAKE_GEN=Visual Studio 15 2017 Win64"  
2020-10-18T20:40:20.9642881Z  set "BITS=64" 
2020-10-18T20:40:20.9643685Z )  else (
2020-10-18T20:40:20.9644404Z set "CMAKE_GEN=Visual Studio 15 2017"  
2020-10-18T20:40:20.9644983Z  set "BITS=32" 
2020-10-18T20:40:20.9645539Z ) 
2020-10-18T20:40:20.9652907Z 
2020-10-18T20:40:20.9654051Z %SRC_DIR%>pushd C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\ 
2020-10-18T20:40:20.9665651Z 
2020-10-18T20:40:20.9668504Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise>CALL "VC\Auxiliary\Build\vcvars64.bat" -vcvars_ver=14.16 10.0.17763.0 
2020-10-18T20:40:21.6606196Z **********************************************************************
2020-10-18T20:40:21.6607656Z ** Visual Studio 2017 Developer Command Prompt v15.9.27
2020-10-18T20:40:21.6608496Z ** Copyright (c) 2017 Microsoft Corporation
2020-10-18T20:40:21.6609621Z **********************************************************************
2020-10-18T20:40:24.0324483Z [vcvarsall.bat] Environment initialized for: 'x64'
2020-10-18T20:40:26.9841640Z -- The C compiler identification is MSVC 19.16.27043.0
2020-10-18T20:40:28.0966868Z -- The CXX compiler identification is MSVC 19.16.27043.0
2020-10-18T20:40:28.1622752Z -- Detecting C compiler ABI info
2020-10-18T20:40:31.0999509Z -- Detecting C compiler ABI info - done
2020-10-18T20:40:31.1040005Z -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Enterprise/VC/Tools/MSVC/14.16.27023/bin/Hostx64/x64/cl.exe - skipped
2020-10-18T20:40:31.1046657Z -- Detecting C compile features
2020-10-18T20:40:31.1058432Z -- Detecting C compile features - done
2020-10-18T20:40:31.1190670Z -- Detecting CXX compiler ABI info
2020-10-18T20:40:32.5928463Z -- Detecting CXX compiler ABI info - done
2020-10-18T20:40:32.5964591Z -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2017/Enterprise/VC/Tools/MSVC/14.16.27023/bin/Hostx64/x64/cl.exe - skipped
2020-10-18T20:40:32.5971494Z -- Detecting CXX compile features
2020-10-18T20:40:32.5985718Z -- Detecting CXX compile features - done
2020-10-18T20:40:38.1882558Z -- [download 0% complete]
2020-10-18T20:40:38.2489267Z -- [download 5% complete]
2020-10-18T20:40:38.3092798Z -- [download 8% complete]
2020-10-18T20:40:38.3094924Z -- [download 11% complete]
2020-10-18T20:40:38.3095395Z -- [download 13% complete]
2020-10-18T20:40:38.3691525Z -- [download 22% complete]
2020-10-18T20:40:38.3699254Z -- [download 24% complete]
2020-10-18T20:40:38.3701968Z -- [download 27% complete]
2020-10-18T20:40:38.3705950Z -- [download 30% complete]
2020-10-18T20:40:38.3709345Z -- [download 32% complete]
2020-10-18T20:40:38.4297385Z -- [download 40% complete]
2020-10-18T20:40:38.4302985Z -- [download 49% complete]
2020-10-18T20:40:38.4309167Z -- [download 59% complete]
2020-10-18T20:40:38.4313713Z -- [download 65% complete]
2020-10-18T20:40:38.4319785Z -- [download 70% complete]
2020-10-18T20:40:38.4911470Z -- [download 100% complete]
2020-10-18T20:40:39.2287423Z -- [download 0% complete]
2020-10-18T20:40:39.3037960Z -- [download 4% complete]
2020-10-18T20:40:39.3789427Z -- [download 11% complete]
2020-10-18T20:40:39.4545484Z -- [download 15% complete]
2020-10-18T20:40:39.5292639Z -- [download 24% complete]
2020-10-18T20:40:39.6058786Z -- [download 31% complete]
2020-10-18T20:40:39.6796275Z -- [download 39% complete]
2020-10-18T20:40:39.7574198Z -- [download 48% complete]
2020-10-18T20:40:39.8339622Z -- [download 57% complete]
2020-10-18T20:40:39.9084039Z -- [download 67% complete]
2020-10-18T20:40:39.9836066Z -- [download 76% complete]
2020-10-18T20:40:40.0599065Z -- [download 85% complete]
2020-10-18T20:40:40.1348500Z -- [download 94% complete]
2020-10-18T20:40:40.2091886Z -- [download 100% complete]
2020-10-18T20:40:40.6830837Z -- Found TIFF: D:/bld/match-series_1603053385828/_h_env/Library/lib/tiff.lib (found version "4.1.0") 
2020-10-18T20:40:40.6949304Z -- Looking for sgemm_
2020-10-18T20:40:41.2520724Z -- Looking for sgemm_ - not found
2020-10-18T20:40:41.2611811Z -- Looking for pthread.h
2020-10-18T20:40:41.6493407Z -- Looking for pthread.h - not found
2020-10-18T20:40:41.6515020Z -- Found Threads: TRUE  
2020-10-18T20:40:42.1495859Z -- Looking for sgemm_
2020-10-18T20:40:44.0199838Z -- Looking for sgemm_ - found
2020-10-18T20:40:44.0212390Z -- Found BLAS: D:/bld/match-series_1603053385828/_h_env/Library/lib/blas.lib  
2020-10-18T20:40:44.0261925Z -- Looking for cheev_
2020-10-18T20:40:44.6504289Z -- Looking for cheev_ - not found
2020-10-18T20:40:44.7397228Z -- Looking for cheev_
2020-10-18T20:40:45.6313123Z -- Looking for cheev_ - found
2020-10-18T20:40:45.6316416Z -- A library with LAPACK API found.
2020-10-18T20:40:45.7055923Z -- Configuring done
2020-10-18T20:40:47.1160389Z -- Generating done
2020-10-18T20:40:47.1459880Z CMake Warning:
2020-10-18T20:40:47.1461016Z   Manually-specified variables were not used by the project:
2020-10-18T20:40:47.1461773Z 
2020-10-18T20:40:47.1462477Z     BUILD_FOX
2020-10-18T20:40:47.1463363Z     GCC_SYSTEM_TYPE
2020-10-18T20:40:47.1464393Z     PARSE_GCC_ERRORS
2020-10-18T20:40:47.1464917Z     TEST_OPENSOURCE_SELECTION
2020-10-18T20:40:47.1465306Z     USE_C++11
2020-10-18T20:40:47.1465589Z     USE_PROFILING
2020-10-18T20:40:47.1466227Z     USE_SSE
2020-10-18T20:40:47.1466562Z 
2020-10-18T20:40:47.1466891Z 
2020-10-18T20:40:47.1479162Z -- Build files have been written to: D:/bld/match-series_1603053385828/work/amd64-pc-windows-19.0.0
2020-10-18T20:40:47.1797449Z 
2020-10-18T20:40:47.1806471Z Microsoft (R) Program Maintenance Utility Version 14.16.27043.0
2020-10-18T20:40:47.1807888Z Copyright (C) Microsoft Corporation.  All rights reserved.
2020-10-18T20:40:47.1808444Z 
2020-10-18T20:40:47.5372135Z Scanning dependencies of target zlib
2020-10-18T20:40:47.5943069Z [  1%] Building C object external/zlib-source/CMakeFiles/zlib.dir/adler32.c.obj
2020-10-18T20:40:47.6250808Z adler32.c
2020-10-18T20:40:47.9378096Z [  2%] Building C object external/zlib-source/CMakeFiles/zlib.dir/compress.c.obj
2020-10-18T20:40:47.9646331Z compress.c
2020-10-18T20:40:48.0404517Z [  3%] Building C object external/zlib-source/CMakeFiles/zlib.dir/crc32.c.obj
2020-10-18T20:40:48.0704411Z crc32.c
2020-10-18T20:40:48.2378638Z [  3%] Building C object external/zlib-source/CMakeFiles/zlib.dir/deflate.c.obj
2020-10-18T20:40:48.2635750Z deflate.c
2020-10-18T20:40:48.6176157Z [  4%] Building C object external/zlib-source/CMakeFiles/zlib.dir/gzclose.c.obj
2020-10-18T20:40:48.6421811Z gzclose.c
2020-10-18T20:40:48.7434358Z [  5%] Building C object external/zlib-source/CMakeFiles/zlib.dir/gzlib.c.obj
2020-10-18T20:40:48.7671267Z gzlib.c
2020-10-18T20:40:48.8845411Z [  6%] Building C object external/zlib-source/CMakeFiles/zlib.dir/gzread.c.obj
2020-10-18T20:40:48.9101185Z gzread.c
2020-10-18T20:40:49.0626480Z [  7%] Building C object external/zlib-source/CMakeFiles/zlib.dir/gzwrite.c.obj
2020-10-18T20:40:49.0848787Z gzwrite.c
2020-10-18T20:40:49.2303808Z [  8%] Building C object external/zlib-source/CMakeFiles/zlib.dir/infback.c.obj
2020-10-18T20:40:49.2614972Z infback.c
2020-10-18T20:40:49.4664648Z [  8%] Building C object external/zlib-source/CMakeFiles/zlib.dir/inffast.c.obj
2020-10-18T20:40:49.4890792Z inffast.c
2020-10-18T20:40:49.6060778Z [  9%] Building C object external/zlib-source/CMakeFiles/zlib.dir/inflate.c.obj
2020-10-18T20:40:49.6341302Z inflate.c
2020-10-18T20:40:49.9809974Z [ 10%] Building C object external/zlib-source/CMakeFiles/zlib.dir/inftrees.c.obj
2020-10-18T20:40:50.0039957Z inftrees.c
2020-10-18T20:40:50.7192190Z [ 11%] Building C object external/zlib-source/CMakeFiles/zlib.dir/trees.c.obj
2020-10-18T20:40:50.7193679Z trees.c
2020-10-18T20:40:50.7194514Z [ 12%] Building C object external/zlib-source/CMakeFiles/zlib.dir/uncompr.c.obj
2020-10-18T20:40:50.7195247Z uncompr.c
2020-10-18T20:40:50.7195725Z [ 13%] Building C object external/zlib-source/CMakeFiles/zlib.dir/zutil.c.obj
2020-10-18T20:40:50.7197685Z zutil.c
2020-10-18T20:40:50.7198102Z [ 14%] Linking C static library zlib.lib
2020-10-18T20:40:50.9042699Z [ 14%] Built target zlib
2020-10-18T20:40:51.0182875Z Scanning dependencies of target png
2020-10-18T20:40:51.0738507Z [ 15%] Building C object external/png-source/CMakeFiles/png.dir/png.c.obj
2020-10-18T20:40:51.1053064Z png.c
2020-10-18T20:40:51.7549587Z [ 16%] Building C object external/png-source/CMakeFiles/png.dir/pngerror.c.obj
2020-10-18T20:40:51.7912398Z pngerror.c
2020-10-18T20:40:52.3602183Z [ 17%] Building C object external/png-source/CMakeFiles/png.dir/pngget.c.obj
2020-10-18T20:40:52.3923907Z pngget.c
2020-10-18T20:40:53.0836378Z [ 18%] Building C object external/png-source/CMakeFiles/png.dir/pngmem.c.obj
2020-10-18T20:40:53.1240685Z pngmem.c
2020-10-18T20:40:53.6850143Z [ 19%] Building C object external/png-source/CMakeFiles/png.dir/pngpread.c.obj
2020-10-18T20:40:53.7237180Z pngpread.c
2020-10-18T20:40:54.3259835Z [ 19%] Building C object external/png-source/CMakeFiles/png.dir/pngread.c.obj
2020-10-18T20:40:54.3528831Z pngread.c
2020-10-18T20:40:54.9670849Z [ 20%] Building C object external/png-source/CMakeFiles/png.dir/pngrio.c.obj
2020-10-18T20:40:54.9973069Z pngrio.c
2020-10-18T20:40:55.4699707Z [ 21%] Building C object external/png-source/CMakeFiles/png.dir/pngrtran.c.obj
2020-10-18T20:40:55.4960081Z pngrtran.c
2020-10-18T20:40:56.5510366Z [ 22%] Building C object external/png-source/CMakeFiles/png.dir/pngrutil.c.obj
2020-10-18T20:40:56.5760338Z pngrutil.c
2020-10-18T20:40:57.3419794Z [ 23%] Building C object external/png-source/CMakeFiles/png.dir/pngset.c.obj
2020-10-18T20:40:57.3715874Z pngset.c
2020-10-18T20:40:57.9887378Z [ 24%] Building C object external/png-source/CMakeFiles/png.dir/pngtrans.c.obj
2020-10-18T20:40:58.0238650Z pngtrans.c
2020-10-18T20:40:58.6314400Z [ 24%] Building C object external/png-source/CMakeFiles/png.dir/pngwio.c.obj
2020-10-18T20:40:58.6669672Z pngwio.c
2020-10-18T20:40:59.2190725Z [ 25%] Building C object external/png-source/CMakeFiles/png.dir/pngwrite.c.obj
2020-10-18T20:40:59.2501937Z pngwrite.c
2020-10-18T20:40:59.8117424Z [ 26%] Building C object external/png-source/CMakeFiles/png.dir/pngwtran.c.obj
2020-10-18T20:40:59.8389757Z pngwtran.c
2020-10-18T20:41:00.4212401Z [ 27%] Building C object external/png-source/CMakeFiles/png.dir/pngwutil.c.obj
2020-10-18T20:41:00.4474033Z pngwutil.c
2020-10-18T20:41:01.2719500Z [ 28%] Linking C static library png.lib
2020-10-18T20:41:01.4205488Z [ 28%] Built target png
2020-10-18T20:41:01.8028231Z Scanning dependencies of target libaol
2020-10-18T20:41:01.9209824Z [ 28%] Building CXX object CMakeFiles/libaol.dir/modules/aol/adaptiveTriangMesh.cpp.obj
2020-10-18T20:41:01.9546373Z adaptiveTriangMesh.cpp
2020-10-18T20:41:02.3662755Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:10.4763921Z [ 29%] Building CXX object CMakeFiles/libaol.dir/modules/aol/aol.cpp.obj
2020-10-18T20:41:10.5181842Z aol.cpp
2020-10-18T20:41:11.5699181Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:12.5174798Z [ 30%] Building CXX object CMakeFiles/libaol.dir/modules/aol/bandMatrix.cpp.obj
2020-10-18T20:41:12.5522886Z bandMatrix.cpp
2020-10-18T20:41:12.8222499Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:15.5184176Z [ 31%] Building CXX object CMakeFiles/libaol.dir/modules/aol/bitVector.cpp.obj
2020-10-18T20:41:15.5660173Z bitVector.cpp
2020-10-18T20:41:15.8835894Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:17.6257434Z [ 32%] Building CXX object CMakeFiles/libaol.dir/modules/aol/ctrlCCatcher.cpp.obj
2020-10-18T20:41:17.6638469Z ctrlCCatcher.cpp
2020-10-18T20:41:17.9850294Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:18.7692682Z [ 33%] Building CXX object CMakeFiles/libaol.dir/modules/aol/diagBandMatrix.cpp.obj
2020-10-18T20:41:18.8006835Z diagBandMatrix.cpp
2020-10-18T20:41:19.1454554Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:21.6451602Z [ 33%] Building CXX object CMakeFiles/libaol.dir/modules/aol/epswriter.cpp.obj
2020-10-18T20:41:21.7111444Z epswriter.cpp
2020-10-18T20:41:22.0096625Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:24.0805151Z [ 34%] Building CXX object CMakeFiles/libaol.dir/modules/aol/geom.cpp.obj
2020-10-18T20:41:24.1132215Z geom.cpp
2020-10-18T20:41:24.4342101Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:26.1268450Z [ 35%] Building CXX object CMakeFiles/libaol.dir/modules/aol/gnuplotter.cpp.obj
2020-10-18T20:41:26.1582673Z gnuplotter.cpp
2020-10-18T20:41:26.3993810Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:28.9135093Z [ 36%] Building CXX object CMakeFiles/libaol.dir/modules/aol/matrix.cpp.obj
2020-10-18T20:41:28.9436421Z matrix.cpp
2020-10-18T20:41:29.2837270Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:37.0448424Z [ 37%] Building CXX object CMakeFiles/libaol.dir/modules/aol/matrixInverse.cpp.obj
2020-10-18T20:41:37.0899600Z matrixInverse.cpp
2020-10-18T20:41:37.4640430Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:39.8460489Z [ 38%] Building CXX object CMakeFiles/libaol.dir/modules/aol/memoryManager.cpp.obj
2020-10-18T20:41:39.8865108Z memoryManager.cpp
2020-10-18T20:41:40.1975781Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:41.3343986Z [ 38%] Building CXX object CMakeFiles/libaol.dir/modules/aol/multiStreambuf.cpp.obj
2020-10-18T20:41:41.3725351Z multiStreambuf.cpp
2020-10-18T20:41:42.1479641Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:43.0688023Z [ 39%] Building CXX object CMakeFiles/libaol.dir/modules/aol/multiVector.cpp.obj
2020-10-18T20:41:43.1220367Z multiVector.cpp
2020-10-18T20:41:43.4642965Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:52.7066067Z [ 40%] Building CXX object CMakeFiles/libaol.dir/modules/aol/parameterParser.cpp.obj
2020-10-18T20:41:52.7425056Z parameterParser.cpp
2020-10-18T20:41:53.0383058Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:55.1174150Z [ 41%] Building CXX object CMakeFiles/libaol.dir/modules/aol/platformDependent.cpp.obj
2020-10-18T20:41:55.1545532Z platformDependent.cpp
2020-10-18T20:41:56.2541707Z C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\include\xlocale(319): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
2020-10-18T20:41:56.6094792Z %SRC_DIR%\quocmesh\modules\aol\platformDependent.cpp(256): error C2664: 'DWORD GetTempPathA(DWORD,LPSTR)': cannot convert argument 2 from 'WCHAR [1024]' to 'LPSTR'
2020-10-18T20:41:56.6097593Z %SRC_DIR%\quocmesh\modules\aol\platformDependent.cpp(257): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style cast
2020-10-18T20:41:56.6102958Z %SRC_DIR%\quocmesh\modules\aol\platformDependent.cpp(265): error C2664: 'UINT GetTempFileNameA(LPCSTR,LPCSTR,UINT,LPSTR)': cannot convert argument 1 from 'WCHAR [1024]' to 'LPCSTR'
2020-10-18T20:41:56.6104688Z %SRC_DIR%\quocmesh\modules\aol\platformDependent.cpp(268): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style cast
2020-10-18T20:41:56.6459857Z NMAKE : fatal error U1077: 'C:\PROGRA~2\MICROS~1\2017\ENTERP~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe' : return code '0x2'
2020-10-18T20:41:56.6462004Z Stop.
2020-10-18T20:41:56.6554745Z NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64\nmake.exe"' : return code '0x2'
2020-10-18T20:41:56.6556390Z Stop.
2020-10-18T20:41:56.6557701Z NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64\nmake.exe"' : return code '0x2'
2020-10-18T20:41:56.6558775Z Stop.
2020-10-18T20:41:57.0293948Z Traceback (most recent call last):
2020-10-18T20:41:57.0295080Z   File "C:\Miniconda\Scripts\conda-build-script.py", line 10, in <module>
2020-10-18T20:41:57.0296934Z     sys.exit(main())
2020-10-18T20:41:57.0297738Z   File "C:\Miniconda\lib\site-packages\conda_build\cli\main_build.py", line 474, in main
2020-10-18T20:41:57.0302647Z     execute(sys.argv[1:])
2020-10-18T20:41:57.0303642Z   File "C:\Miniconda\lib\site-packages\conda_build\cli\main_build.py", line 465, in execute
2020-10-18T20:41:57.0305441Z     verify=args.verify, variants=args.variants)
2020-10-18T20:41:57.0306344Z   File "C:\Miniconda\lib\site-packages\conda_build\api.py", line 195, in build
2020-10-18T20:41:57.0308723Z     variants=variants
2020-10-18T20:41:57.0309383Z   File "C:\Miniconda\lib\site-packages\conda_build\build.py", line 3013, in build_tree
2020-10-18T20:41:57.0322723Z     notest=notest,
2020-10-18T20:41:57.0323592Z   File "C:\Miniconda\lib\site-packages\conda_build\build.py", line 2099, in build
2020-10-18T20:41:57.0335255Z     windows.build(m, build_file, stats=build_stats, provision_only=provision_only)
2020-10-18T20:41:57.0336308Z   File "C:\Miniconda\lib\site-packages\conda_build\windows.py", line 299, in build
2020-10-18T20:41:57.0338138Z     check_call_env(cmd, cwd=m.config.work_dir, stats=stats, rewrite_stdout_env=rewrite_env)
2020-10-18T20:41:57.0339164Z   File "C:\Miniconda\lib\site-packages\conda_build\utils.py", line 410, in check_call_env
2020-10-18T20:41:57.0341781Z     return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
2020-10-18T20:41:57.0344539Z   File "C:\Miniconda\lib\site-packages\conda_build\utils.py", line 390, in _func_defaulting_env_to_os_environ
2020-10-18T20:41:57.0346850Z     raise subprocess.CalledProcessError(proc.returncode, _args)
2020-10-18T20:41:57.0348271Z subprocess.CalledProcessError: Command '['cmd.exe', '/d', '/c', 'conda_build.bat']' returned non-zero exit status 1.
2020-10-18T20:42:00.9835576Z ##[error]Cmd.exe exited with code '1'.
2020-10-18T20:42:01.0893730Z ##[section]Finishing: Build recipe
```
</p></details>

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -1,6 +1,6 @@
 mkdir %build% && cd %build%
 
-cmake -G"NMake Makefiles" ^
+cmake -G"MinGW Makefiles" ^
       -DCMAKE_INSTALL_PREFIX=%PREFIX% ^
       -DCMAKE_CXX_FLAGS=%CXXFLAGS% ^
       -DCMAKE_C_FLAGS=%CFLAGS% ^
@@ -9,20 +9,20 @@ cmake -G"NMake Makefiles" ^
       -DUSE_PNG=1 ^
       -DPARSE_GCC_ERRORS=0 ^
       -DUSE_OPENMP=0 ^
-      -DDYNAMIC_LINKING=0 ^
+      -DDYNAMIC_LINKING=1 ^
       -DUSE_PROFILING=0 ^
-      -DUSE_SSE=0 ^
-      -DUSE_BLAS=0 ^
+      -DUSE_SSE=1 ^
+      -DUSE_BLAS=1 ^
       -DUSE_LAPACK=1 ^
       -DUSE_DOXYGEN=0 ^
       -DUSE_QT=0 ^
       -DUSE_CUDA=0 ^
       -DUSE_OPENMESH=0 ^
       -DUSE_VTK=0 ^
-      -DUSE_SUITESPARSE=0 ^
+      -DUSE_SUITESPARSE=1 ^
       -DUSE_FOX=0 ^
       -DBUILD_FOX=0 ^
-      -DUSE_FFTW=0 ^
+      -DUSE_FFTW=1 ^
       -DBUILD_ZLIB_AND_LIBPNG=1 ^
       -DUSE_BOOST=0 ^
       -DUSE_C++11=1 ^

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -19,7 +19,7 @@ cmake -G"NMake Makefiles" ^
       -DUSE_CUDA=0 ^
       -DUSE_OPENMESH=0 ^
       -DUSE_VTK=0 ^
-      -DUSE_SUITESPARSE=1 ^
+      -DUSE_SUITESPARSE=0 ^
       -DUSE_FOX=0 ^
       -DBUILD_FOX=0 ^
       -DUSE_FFTW=1 ^

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -1,6 +1,6 @@
 mkdir %build% && cd %build%
 
-cmake -G"MinGW Makefiles" ^
+cmake -G"NMake Makefiles" ^
       -DCMAKE_INSTALL_PREFIX=%PREFIX% ^
       -DCMAKE_CXX_FLAGS=%CXXFLAGS% ^
       -DCMAKE_C_FLAGS=%CFLAGS% ^

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -7,7 +7,8 @@ package:
 
 source:
   url: https://github.com/berkels/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 9e74ca6bc58571065f30f99307f8912009fa6b62be10f24d3f5ddc5b2361ce48
+  url: https://github.com/berkels/match-series/archive/master.zip
+  sha256: 5c09e8896547815fd5e9496d218667327a1943f720817dbf161a9c220c87a472
 
 build:
   number: 6

